### ISSUE TITLE
Autoimport breaks positional elements (module docstring, shebang, file variables, ...)

### ISSUE THREAD

**Issue by kbauer** [lyz-code/autoimport#70]
## Description

When used on a python script, `autoimport` breaks the shebang and the file-variables line used by Emacs (which must come immediately after the shebang).

## Steps to reproduce
Create a script file `test.py` with contents

```
#!/usr/bin/env python3
# -*- coding: latin-1 -*-
"""docstring"""
print(os.path.exists("."))
```
and run `autoimport test.py`.

## Current behavior
The file will now contain
```
import os

#!/usr/bin/env python3
# -*- coding: latin-1 -*-
"""docstring"""
print(os.path.exists("."))
```

Not only does this break the shebang, module docstring and file variables, it is also potentially dangerous, because `import` is an existing executable associated with ImageMagick.

## Desired behavior
There are many possible ways changing the position of the imports might break a python file, some of which might depend on the tooling used by the developer.

Imports should therefore be positioned

  1. After all leading comment lines,
  2. At the position where imports are currently located.

## Environment

           autoimport version: 0.5.0
               python version: 3.8.5 (default, Sep  4 2020, 07:30:14)  [GCC 7.3.0]
                     platform: Linux-4.12.14-lp151.28.52-default-x86_64-with-glibc2.10

**Comment by lyz-code** [comment]
Hi @kbauer thanks for opening the issue, 

I've solved it in #71 and released under version 0.6.0. 

Cheers

**Comment by kbauer** [comment]
@lyz-code That was fast :)

However the fix might make too many assumptions. E.g.:

```
#!/usr/bin/env python3
# -*- coding: latin-1 -*-

# pylint: disable=foobar

"""

This is the docstring.

"""

import sys

print(os.path.exists(sys.argv[1]))
```
becomes
```
#!/usr/bin/env python3
# -*- coding: latin-1 -*-

import sys
import os

# pylint: disable=foobar

"""

This is the docstring.

"""


print(os.path.exists(sys.argv[1]))
```
Diff:
![image](https://user-images.githubusercontent.com/7498468/106461713-6efe5b00-6495-11eb-8719-c70b9f7c7c22.png)

Apparently, `autoimport` also removes the trailing newline, which `git` isn't happy about.

**Comment by lyz-code** [comment]
Hi, I see your points, I'll fix it tomorrow.

Regarding the trailing newline, I did it to be compliant with the [pre-commit hook](https://github.com/pre-commit/pre-commit-hooks#trailing-whitespace), but I understand that, it's something that should be handled by other tools, so I'll respect trailing newlines at the end of the file.

Thanks!

**Comment by lyz-code** [comment]
@kbauer Fixed in #72 and released under `0.6.1`. If you find any other case, do not hesitate to reopen it
